### ISSUE TITLE
Log4J message

### ISSUE THREAD

**Issue by David-Brabant** [scottlowe/CuratorIkvm#1]
To fix the log4j message appearing when you start a console application using your package, you can add the following line in pom.xml, before the dependency to log4j-slf4j-impl:

```
    <dependency>
        <groupId>org.apache.logging.log4j</groupId>
        <artifactId>log4j-core</artifactId>
      <version>2.1</version>
    </dependency>
```

then add the following line in the appSettings of your application config file:

```
<add key="ikvm:log4j.configurationFile" value="log4j2.xml" />
```

and finally, place a file named log4j2xml (such as the one below) in your executable path:

```
<?xml version="1.0" encoding="UTF-8"?>
<Configuration status="WARN">
  <Appenders>
    <Console name="Console" target="SYSTEM_OUT">
      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
    </Console>
  </Appenders>
  <Loggers>
    <Root level="error">
      <AppenderRef ref="Console"/>
    </Root>
  </Loggers>
</Configuration>
```

**Comment by scottlowe** [comment]
Thanks David,

I was very much wanting to fix that, and of course it is annoying for anybody else that is using this code!

**Comment by scottlowe** [comment]
@David-Brabant 

Hi David. I have applied `org.apache.logging.log4j` dependency to `pom.xml` and have committed that to master.

I'm sorry that it's taken so long to do this. I moved back to the UK from Australia where I had a Windows desktop, and haven't had access to a Windows dev machine since. I have finally built one that runs under Parallels on my MacBook.

I also updated IKVM and fixed paths for Mono compilation on Unix machines, and it builds on my Mac, but wanted to be sure that the released version was compiled under Windows until I can be sure that everything is okay.
